.github/workflows/ci-testing.yml

```diff
@@ -0,0 +1,31 @@
+name: Continuous testing & Linting
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Do you have a particular reason why testing against all of those interpreter versions ? If not and my assumption is that this version of rootgrow is a suse only thing, 3.6 (SLE) and 3.8 (TW) should be enough or am I mistaken ? Overall I would only test for interpreters where we also have integration tests or at least real-use scenarios

Reply:
Well 3.9 is coming to TW I think it is good to know ahead of time if things start failing.

Reply:
I like to test all active versions. There's always a chance something changes that breaks a project. Agree with Robert on catching issues early.